server/pom.xml

```diff
@@ -121,6 +121,11 @@
             <version>4.5.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
We should really try to keep the dependencies organized, otherwise this will look horrible at the end.